pytoda/proteins/transforms.py

```diff
@@ -27,3 +34,310 @@ def __call__(self, smiles: str) -> Indexes:
             Indexes: indexes representation for the Sequence provided.
         """
         return self.protein_language.sequence_to_token_indexes(smiles)
+
+
+class ReplaceByFullProteinSequence(Transform):
```

Comment:
is the output expected to be high-case in "residues of interest" and lower-case outside?
if yes I think it's worth mentioning it in the docstring

Reply:
Done